src/components/select/select.js

```diff
@@ -0,0 +1,321 @@
+// @flow
+import * as React from 'react';
+import {findDOMNode} from 'react-dom';
+import {
+  Root as StyledRoot,
+  Input as StyledInput,
+  InputContainer as StyledInputContainer,
+  Tag as StyledTag,
+  SearchIcon as StyledSearchIcon,
+} from './styled-components';
+
+import {Input as InputComponent} from '../input';
+import {ICON, TYPE, STATE_TYPE} from './constants';
+import SelectDropDown from './dropdown';
+import type {
+  DefaultPropsT,
+  LabelT,
+  OptionT,
+  PropsT,
+  StatelessStateT,
+  ChangeActionT,
+} from './types';
+import {getOverride} from '../../helpers/overrides';
+import document from 'global/document';
```

Comment:
This package is entirely bloat assuming `document` is never used outside of browser-only React lifeycle methods, which seems to be the case.

Reply:
what's the solution? We have lint complaining about inline requires and no approach to have bundle based on browser\server code